docs/setup-cluster/elasticsearch-logging-backend.rst

```diff
@@ -4,24 +4,22 @@
  Elasticsearch-backed logging
 ##############################
 
-This guide covers the limitations of the default logging backend, as a guideline on when to migrate
-to Elasticsearch, and some tips for how to tune Elasticsearch to work best with Determined.
+It is possible to use Elasticsearch as an alternative to the default logging backend. In the past,
+such configuration was recommended for larger installations, but it is no longer the case.
```

Comment:
It is possible to use Elasticsearch as an alternative to the default logging backend.

(remove the second sentence)